pkg/services/ngalert/api/test-data/post-rulegroup-101-export.hcl

```diff
@@ -17,7 +17,7 @@ resource "grafana_rule_group" "rule_group_0000" {
       }
 
       datasource_uid = "000000002"
-      model          = "{\n              \"expr\": \"http_request_duration_microseconds_count\",\n              \"hide\": false,\n              \"interval\": \"\",\n              \"intervalMs\": 1000,\n              \"legendFormat\": \"\",\n              \"maxDataPoints\": 100,\n              \"refId\": \"query\"\n            }"
```

Comment:
I wonder if it's possible to keep the result the same. Changes in export could cause diff downstream.

Reply:
Considering this also removes the noise coming from the extra whitespace and newlines, I'm of the opinion that it's not worth spending time working to keep the current output. WDYT? 

That being said if we feel like consistency is more important I would have to imagine there's a way to maintain the formatting.

Reply:
It was determined this was a quirk of the test. I've pushed up a change that compacts the json to make the behavior clearer, but in any case rules previously exported like this should not contain those whitespace characters.